docs/manifest.json

```diff
@@ -534,10 +541,10 @@
 			]
 		},
 		{
-			"title": "Enterprise",
-			"description": "Learn how to enable Enterprise features",
-			"path": "./enterprise.md",
-			"icon_path": "./images/icons/group.svg"
+			"title": "Licensing",
+			"description": "Learn how to enable Premium features",
+			"path": "./licensing.md",
+			"icon_path": "./images/icons/enterprise.svg"
```

Comment:
nit: should make an issue to rename this icon.

Reply:
Will rename!